src/ucp/proto/proto_am.inl

```diff
@@ -434,7 +436,17 @@ ucs_status_t ucp_do_am_zcopy_multi(uct_pending_req_t *self, uint8_t am_id_first,
 
     max_zcopy = ucp_ep_get_max_zcopy(ep, req->send.lane) - user_hdr_size;
     max_iov   = ucp_ep_get_max_iov(ep, req->send.lane) - !!user_hdr_size;
-    iov       = ucs_alloca((max_iov + 1) * sizeof(uct_iov_t));
+    iov_size  = (max_iov + 1) * sizeof(uct_iov_t);
```

Comment:
maybe just limit max_iov by UCS_ALLOCA_MAX_SIZE?

`max_iov = ucs_min(ucp_ep_get_max_iov(...), UCS_ALLOCA_MAX_SIZE / sizeof(*iov))`

Reply:
yes
BTW, which test case failed? can we add gtest for it?

Reply:
I added the testcase in this PR, `send_process_iov_am_64k_size` can reproduce the issue.

Reply:
@yosefe please review the new commits. Thanks